codemeta.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
If we plan on adding `codemeta.json` then we will also need to include a `.gitignore` for packaged releases. 🤔
https://docs.github.com/en/get-started/git-basics/ignoring-files

Reply:
Is this something I can do on my side?

Reply:
I've changed the branch to develop which includes this already.